app/src/common/shared/org/mozilla/vrbrowser/VRBrowserActivity.java

```diff
@@ -1028,5 +1036,6 @@ public void setCylinderDensity(final float aDensity) {
     private native void setControllersVisibleNative(boolean aVisible);
     private native void runCallbackNative(long aCallback);
     private native void setCylinderDensityNative(float aDensity);
+    private native void setCPULevelNative(boolean aHighLevel);
```

Comment:
One nit I have is I would like to move away from using booleans as arguments as they can make the call site inscrutable without looking at the function implementation. I realize Java doesn't have any really good solutions for the JNI boundary but I sort of feel like the extra work to use integer constants helps improve code readability. @MortimerGoro What are your thoughts on this?

Reply:
I think it's a good practice. I'll update the PR.